crates/sui-types/src/sui_system_state/epoch_start_sui_system_state.rs

```diff
@@ -198,10 +197,20 @@ impl EpochStartSystemStateTrait for EpochStartSystemStateV1 {
             });
         }
 
-        // Sort the authorities by their protocol (public) key in ascending order. That's the sorting
-        // SUI committee follows and we
+        // Sort the authorities by their protocol (public) key in ascending order, same as the order
+        // in the Sui committee returned from get_sui_committee().
         authorities.sort_by(|a1, a2| a1.protocol_key.cmp(&a2.protocol_key));
 
+        for ((sui_authority_name, _), mysticeti_authority) in
+            self.get_sui_committee().members().zip(authorities.iter())
+        {
+            if sui_authority_name
+                != &AuthorityPublicKeyBytes::from(&mysticeti_authority.protocol_key)
+            {
+                error!("Mismatched Sui and Mysticeti committee! Sui authority: {:?} Mysticeti authority: {:?}", sui_authority_name, mysticeti_authority);
```

Comment:
nit: would it be more accurate to say there is a mismatch between the active validator set and the sui committee? Just so anyone debugging doesn't think it has something to do with mysticeti

Reply:
Not sure if active validator set is different in concept from Sui committee. I will reword this.